calico/getting-started/kubernetes/quickstart.mdx

```diff
@@ -1,170 +1,500 @@
+   calico-apiserver     calico-apiserver-64ff5c4964-4c498                      1/1     Running   0          4m15s
+   calico-apiserver     calico-apiserver-64ff5c4964-k744k                      1/1     Running   0          4m15s
+   calico-system        calico-kube-controllers-788444bfcc-2h4vg               1/1     Running   0          4m11s
+   calico-system        calico-node-gs985                                      1/1     Running   0          4m11s
+   calico-system        calico-node-jx7ts                                      1/1     Running   0          4m11s
+   calico-system        calico-node-mzxp9                                      1/1     Running   0          4m11s
+   calico-system        calico-typha-69c7d597c-fdbnj                           1/1     Running   0          4m4s
+   calico-system        calico-typha-69c7d597c-gtlbl                           1/1     Running   0          4m12s
+   calico-system        csi-node-driver-5msrb                                  2/2     Running   0          4m11s
+   calico-system        csi-node-driver-5zd8b                                  2/2     Running   0          4m11s
+   calico-system        csi-node-driver-gbrvs                                  2/2     Running   0          4m11s
+   calico-system        goldmane-75465fb67d-hwcmr                              1/1     Running   0          4m12s
+   calico-system        whisker-6dc468dd54-pdxkq                               2/2     Running   0          4m12s
+   kube-system          coredns-76f75df574-gnfhr                               1/1     Running   0          32m
+   kube-system          coredns-76f75df574-qh6jh                               1/1     Running   0          32m
+   kube-system          etcd-calico-cluster-control-plane                      1/1     Running   0          32m
+   kube-system          kube-apiserver-calico-cluster-control-plane            1/1     Running   0          32m
+   kube-system          kube-controller-manager-calico-cluster-control-plane   1/1     Running   0          32m
+   kube-system          kube-proxy-898gh                                       1/1     Running   0          32m
+   kube-system          kube-proxy-w9ptk                                       1/1     Running   0          32m
+   kube-system          kube-proxy-wbhpd                                       1/1     Running   0          32m
+   kube-system          kube-scheduler-calico-cluster-control-plane            1/1     Running   0          32m
+   local-path-storage   local-path-provisioner-7577fdbbfb-t4cq7                1/1     Running   0          32m
+   tigera-operator      tigera-operator-576ccbdb88-d9bkm                       1/1     Running   0          6m4s
    ```
 
-### Install $[prodname]
+## Step 3. Monitor network traffic in Calico Whisker
 
-1. Install the Tigera operator and custom resource definitions.
+The Whisker web console deploys automatically, but it is not accessible from outside the cluster.
```

Comment:
Maybe add why its not accessible? 
As a beginner it looks like you are daring me to publish it :D

Reply:
Sounds reasonable. What would you say? Anything more exciting that a generic 'for security reasons'?

Reply:
That is pretty much what I had in mind too: 
> for security reasons since network flows are sensetive infomraiton

Reply:
Added some clarification, thanks.